ruby_enumerable_methods.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    for item in self do
+      yield(item)
+    end
+  end
+
+  def my_each_with_index
+    i = 0
+    for item in self do
+      yield(item, i)
+      i += 1
+    end
+  end
```

Comment:
- enumerables my_each_with_index returns an enumerator if no block is given
     Failure/Error: expect(array.my_each_with_index).to be_an(Enumerator)